gap/semigroups/semitrans.gi

```diff
@@ -735,6 +735,47 @@ function(S)
   return t;
 end);
 
+InstallMethod(WreathProduct, "for a monoid and a permutation group",
```

Comment:
The string here is wrong, it should describe the arguments in the next line. So in this case it should be `for a transformation semigroup and a permutation group`.